purchase_discount/tests/test_purchase_discount.py

```diff
@@ -21,13 +21,17 @@ def setUpClass(cls):
         po_model = cls.env["purchase.order.line"]
         # Set the Exchange rate for the currency of the company to 1
         # to avoid issues with rates
-        cls.env["res.currency.rate"].create(
-            {
-                "currency_id": cls.env.user.company_id.currency_id.id,
-                "rate": 1.00,
-                "name": fields.Date.today(),
-            }
-        )
+        latest_currency_rate_line = currency_rate_model.search([
```

Comment:
No `currency_rate_model` defined before

Reply:
Bad copy-paste, Fixing now

Reply:
Fixed, 
(PS: Wondering why it was not fixed earlier :thinking: )